packages/tpetra/core/test/FECrsMatrix/FECrsMatrix_UnitTests.cpp

```diff
@@ -349,7 +352,8 @@ TEUCHOS_UNIT_TEST_TEMPLATE_4_DECL( FECrsMatrix, Assemble1D_Kokkos, LO, GO, Scala
   auto localMap = pack.overlapMap->getLocalMap();
   //get local map too
 
-  Kokkos::parallel_for(Kokkos::RangePolicy<typename Node::execution_space>(0,k_e2n.extent(0)), 
+  Kokkos::parallel_for("assemble_1d",
+		       range_type (0,k_e2n.extent(0)), 
 		       KOKKOS_LAMBDA(const size_t& i) {
```

Comment:
Best practice is to pass the loop index (or any type that fits in 64 bits) by value, rather than by reference.

Reply:
Thanks.  The kokkos examples are somewhat varied on that.  I've corrected it.